app/models/user.rb

```diff
@@ -47,9 +47,13 @@ class User < ActiveRecord::Base
   scope :reviewers, -> { where(roles: {name: "Reviewer"}).joins(:roles) }
   scope :supervisors, -> { where(roles: {name: "Supervisor"}).joins(:roles) }
   scope :order_fulfilment, -> { where(roles: {name: "Order fulfilment"}).joins(:roles) }
+  scope :order_administrator, -> { where(roles: { name: 'Order administrator' }).joins(:roles) }
   scope :system, -> { where(roles: {name: "System"}).joins(:roles) }
   scope :staff, -> { where(roles: {name: ["Supervisor", "Reviewer"]}).joins(:roles) }
   scope :except_stockit_user, -> { where.not(first_name: "Stockit", last_name: "User") }
+  scope :active, -> { where(disabled: false) }
+  scope :exclude_current_user, ->(id) { where.not(id: id) }
```

Comment:
By changing the name, the method sounds more generically useful.

```suggestion
  scope :exclude_users, ->(id) { where.not(id: id) }
```